cmake/linker_script/common/common-rom.cmake

```diff
@@ -95,6 +95,7 @@ if(CONFIG_NET_SOCKETS)
   zephyr_iterable_section(NAME net_socket_register KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN ${CONFIG_LINKER_ITERABLE_SUBALIGN})
 endif()
 
+zephyr_iterable_section(NAME net_l3_register KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN ${CONFIG_LINKER_ITERABLE_SUBALIGN})
```

Comment:
Shouldn't this be hidden behind `if(CONFIG_NETWORKING)` or similar?

Reply:
Yes, I added the check